docs/quickstart/run_example_quickstart.md

```diff
@@ -125,21 +125,21 @@ public class WikipediaAnalysis {
 }
 {% endhighlight %}
 
-I admit it's very bare bones now but we will fill it as we go. Note, that I'll not give
+I admit it's very bare bones now but we will fill it as we go. Note that I'll not give
```

Comment:
"bare bones" -> "basic"? Or perhaps "The program is very basic now but we will fill it in ..."